packages/lit-dev-content/samples/tutorials/working-with-lists/02/after/my-element.ts

```diff
@@ -0,0 +1,49 @@
+import {LitElement, html} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
+
+@customElement('my-element')
+class MyElement extends LitElement {
+  @property({attribute: false})
+  members = ['Peter', 'Lois', 'Meg', 'Chris', 'Stewie'];
+
+  @property({attribute: false})
+  pets = ['Brian'];
+
+  @state()
+  includePets = true;
+
+  @state()
+  includeSeparator = true;
+
+  render() {
+    const templates = [];
+    for (const member of this.members) {
+      templates.push(html`<li>${member}</li>`);
+    }
+    if (this.includeSeparator) {
+      templates.push('🐶 Pets 🐱');
+    }
+    if (this.includePets) {
+      for (const pet of this.pets) {
+        templates.push(html`<li>${pet}</li>`);
+      }
+    }
+
+    return html`
+      <button @click=${() => this._togglePetVisibility()}>${this.includePets ? 'Hide' : 'Show'} pets</button>
```

Comment:
This looks messy in JS it looks like this:

```ts
return html`
   <button @click=${() => this._togglePetVisibility()}>${
   this.includePets ? 'Hide' : 'Show'
 } pets</button>
   <button @click=${() => this._toggleSeparator()}>${
   this.includeSeparator ? 'Hide' : 'Show'
 } separator</button>
   <p>My family</p>
   <ul>
     <!-- TODO: Render templates. -->
   </ul>
 `;
```

Maybe put the button content on a newline like:

```ts
html`
<button @click=${() => this._togglePetVisibility()}>
  ${this.includePets ? 'Hide' : 'Show'} pets
</button>
...
`
```